src/workloads/scale/LLTInsert.yml

```diff
@@ -112,6 +112,18 @@ Actors:
       PhaseConfig:
         Repeat: 1
 
+# Guard Against timeout for no output.
+- Name: LoggingActor
+  Type: LoggingActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0, 4, 9, 14, 19]
```

Comment:
Is there anyway of having an actor run in all phases without having to list them?